### ISSUE TITLE
emotes cause client crash

### ISSUE THREAD

**Issue by dagit** [matterhorn-chat/matterhorn#163]
Typing `/me tried again` causes a client crash with the following message:

~~~
matterhorn: JSONDecodeException {jsonDecodeExceptionMsg = "mmGetJSONBody: Error in $: endOfInput", jsonDecodeExceptionJSON = "{\"response_type\":\"in_channel\",\"text\":\"*tries again*\",\"username\":\"\",\"icon_url\":\"\",\"goto_location\":\"\",\"attachments\":null}{\"id\":\"api.command.execute_command.save.app_error\",\"message\":\"An error while saving the command response to the channel\",\"detailed_error\":\"\",\"request_id\":\"ics1b7ft3tr87ep5g4hhyyr5qy\",\"status_code\":500}"}
~~~

It seems there are likely two bugs here:
  * The server doesn't like the way we are sending emotes
  * We are not parsing the server response correctly

**Comment by jtdaugherty** [comment]
It turns out that in this case the server response is *invalid* - it has two JSON objects in it, so the parse fails.

**Comment by dagit** [comment]
We've run into that with the MM server before. It seems to be motivated by a limitation of the way their JSON serialization is implemented.

**Comment by jtdaugherty** [comment]
This is now fixed in `30600.1.0`.
